Packs/Illumio/Integrations/IllumioCore/IllumioCore_test.py

```diff
@@ -38,3 +55,224 @@ def test_test_module(requests_mock, mock_client):
     requests_mock.get("https://127.0.0.1:8443/api/v2/health",
                       status_code=200, json={})
     assert test_module(mock_client) == "ok"
+
+
+@pytest.mark.parametrize("args, err_msg", [
+    ({"port": "", "protocol": "tcp", 'start_time': '2022-07-17T12:58:33.528Z', 'end_time': "2022-07-18T12:58:33.529Z",
+      'policy_decisions': ['potentially_blocked']},
+     "{} is a required parameter. Please provide correct value.".format('port')),
+    ({"port": "dummy", "protocol": "tcp", 'start_time': '2022-07-17T12:58:33.528Z',
+      'end_time': "2022-07-18T12:58:33.529Z", 'policy_decisions': ['potentially_blocked']},
+     '"dummy" is not a valid number'),
+    ({"port": -300, "protocol": "tcp", 'start_time': '2022-07-17T12:58:33.528Z',
+      'end_time': "2022-07-18T12:58:33.529Z", 'policy_decisions': ['potentially_blocked']},
+     "{} is an invalid value for port. Value must be in 1 to 65535.".format(-300)),
```

Comment:
You have constants above for these messages, why not use them?

Reply:
Yes, I'll do that

Reply:
Made the changes.